packages/gensx/src/commands/deploy.ts

```diff
@@ -79,6 +79,10 @@ export async function deploy(file: string, options: DeployOptions) {
 
     form.append("schemas", JSON.stringify(schemas));
 
+    if (projectConfig?.description) {
+      form.append("description", projectConfig.description);
```

Comment:
Do we need to update the API?

Reply:
yes, there is a sister PR for the API here: https://github.com/gensx-inc/gensx-console/pull/254/files